src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistorySearch.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import { useSelector, useDispatch } from 'react-redux';
+import { Page, collectionSelectorHelpers, Search } from '@linn-it/linn-form-components-library';
+import bomVerificationHistoryEntriesSearchActions from '../../actions/bomVerificationHistoryEntriesActions';
+import history from '../../history';
+import config from '../../config';
+import { bomVerificationHistoryEntries as bomVerificationHistoryEntriesItemType } from '../../itemTypes';
+
+function BomVerificationHistorySearch() {
+    const dispatch = useDispatch();
+
+    const [searchTerm, setSearchTerm] = useState('');
+
+    const searchEntries = term => dispatch(bomVerificationHistoryEntriesSearchActions.search(term));
+    const clearSearch = () => dispatch(bomVerificationHistoryEntriesSearchActions.clearSearch());
+
+    const searchResults = useSelector(state =>
+        collectionSelectorHelpers.getSearchItems(
+            state[bomVerificationHistoryEntriesItemType.item],
+            100,
+            'tRef',
+            'partNumber',
+            'remarks'
+        )
+    );
+    const searchLoading = useSelector(state =>
+        collectionSelectorHelpers.getSearchLoading(state.bomVerificationHistoryEntries)
+    );
+
+    const goToSelectedEntry = selectedEntry => {
+        history.push(`${bomVerificationHistoryEntriesItemType.uri}/${selectedEntry.tRef}`);
+    };
+
+    const goToCreate = () => {
+        history.push(`${bomVerificationHistoryEntriesItemType.uri}/create`);
```

Comment:
could also possibly add a rel: create link to the resource for this one, for niceness